src/test/java/com/dawn/jat/illuminati/post/advice/PostExceptionAdviceTest.java

```diff
@@ -16,9 +16,15 @@
 
     @Test
     void exception_throwException_returnHttpPostNotFound() {
-        PostNotFoundException postException = new PostNotFoundException("");
-        ResponseEntity<Object> rsp = advice.exception(postException);
+        PostNotFoundException postException = new PostNotFoundException("Cannot find post");
+
+        ResponseEntity<ErrorResponse> rsp = advice.exception(postException);
+        assertEquals("Cannot find post", postException.getMessage());
         assertEquals(HttpStatus.NOT_FOUND, rsp.getStatusCode());
+
+        ErrorResponse errResp = new ErrorResponse(HttpStatus.NO_CONTENT, postException);
```

Comment:
is this test case for `ErrorResponse`, why do we put it here?

Reply:
Updated